service/src/test/java/tds/assessment/builders/AssessmentRecordBuilder.java

```diff
@@ -0,0 +1,210 @@
+package tds.assessment.builders;
+
+import java.sql.Timestamp;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import tds.assessment.Algorithm;
+
+/**
+ * Build data structures that represent what Assessment data looks like in the database.  This class is intended tu
+ * support testing the {@code AssessmentMapper}.
+ */
+public class AssessmentRecordBuilder {
+    private static Timestamp segmentFieldTestStartDate;
```

Comment:
Why are these all static?  That means that once set the values are tied to the class itself.  So if one test modifies the value of `segmentFieldTestStartDate` it could impact another test.